cms/static/js/spec/views/login_studio_spec.js

```diff
@@ -0,0 +1,14 @@
+define(['jquery'],
+function($) {
+    'use strict';
+    describe("Studio Login Page", function() {
+
+        beforeEach(function() {
+            loadFixtures('login.html');
```

Comment:
It looks like other tests do `setFixtures(readFixtures`: https://github.com/edx/edx-platform/blob/master/cms/static/js/spec/utils/drag_and_drop_spec.js#L5. Maybe you can try that?